ops/opset13/abs.go

```diff
@@ -6,6 +6,14 @@ import (
 	"gorgonia.org/tensor"
 )
 
+const (
+	// MinAbsInput is the minimimum amount of inputs the abs operator expects.
+	MinAbsInput = 1
+
+	// MaxAbsInput is the maximum amount of inputs the abs operator accepts.
+	MaxAbsInput = 1
```

Comment:
We don't have comments for all these yet. Do we really want these comments? They seem trivial to me.